to fix: #include <errno.h> in src/hx/thread/CountingSemaphore.posix.cpp

android-ndk-r21e
hxcpp: 4.3.2 [git]
haxe --version
4.3.7



error message:
```
src/hx/thread/CountingSema

phore.posix.cpp:81:8: error: use of undeclared identifier 'errno'

                       if (errno == ETIMEDOUT)
```


to fix, add "#include <errno.h>" in:

```
hxcpp/git/src/hx/thread/CountingSemaphore.posix.cpp
```

--- a/src/hx/thread/CountingSemaphore.posix.cpp
+++ b/src/hx/thread/CountingSemaphore.posix.cpp
@@ -2,6 +2,7 @@
 #include <semaphore.h>
 #include <sys/time.h>
 #include <memory>
+#include <cerrno>
 #include <hx/thread/CountingSemaphore.hpp>
 
 struct hx::thread::CountingSemaphore_obj::Impl

